index.js

```diff
@@ -1,13 +1,14 @@
 var relative = require('relative-date')
 var pretty = require('prettier-bytes')
 var path = require('path')
-var data = require('render-data')
 var yo = require('yo-yo')
+var data = module.parent ? null : require('render-data')
 
 module.exports = Tree
 
 function Tree (root, entries, onclick) {
   if (!(this instanceof Tree)) return new Tree(root, entries, onclick)
+  if (module.parent) onclick = function () {}
```

Comment:
When you're client side, `module.parent` can or should still exist, so this is not a reliable check. I'd rather check `if (process.browser)` which is only true when bundled through browserify.